packages/web/src/pages/search-page-v2/search-results/TrackResults.tsx

```diff
@@ -89,11 +91,15 @@ export const TrackResults = (props: TrackResultsProps) => {
 
   useDebounce(
     () => {
+      // Reuse the existing lineup if the search params haven't changed
+      if (searchParams === prevSearchParams) {
+        return
+      }
       dispatch(searchResultsPageTracksLineupActions.reset())
-      getResults(0, 10, true)
+      getResults(0, 12, true)
```

Comment:
magic number

Reply:
think this is just the limit, but yea it would be nice to label this